egs/swbd/s5c/local/chain/tuning/run_tdnn_7i.sh

```diff
@@ -1,24 +1,25 @@
 #!/bin/bash
-
-# Same as 7h but double the number of parameters (27983950 vs 15551509)
-
+# _7i is as _7h but it uses random CMN offsets generated using between-speaker covariance
```

Comment:
Don't replace 7i if it exists. Create a new one, the next index that does not exist.